.tool-versions

```diff
@@ -1,2 +1,2 @@
 ruby 2.7.1
-nodejs 12.18.3
+nodejs 14.13.0
```

Comment:
Can we stay at 12? 14 is not LTS and can introduce weird/breaking changes

Reply:
it didn't as far as I've tested it, but we can go with active lts I guess :)